benchmark/benchmarks.jl

```diff
@@ -1,18 +1,37 @@
 using BenchmarkTools
 using Graphs
 
-DIGRAPHS = Dict{String,DiGraph}(
+const DIGRAPHS = Dict{String,DiGraph}(
     "complete100" => complete_digraph(100), "path500" => path_digraph(500)
 )
 
-GRAPHS = Dict{String,Graph}(
+const GRAPHS = Dict{String,Graph}(
     "complete100" => complete_graph(100),
     "tutte" => smallgraph(:tutte),
     "path500" => path_graph(500),
 )
 
-suite = BenchmarkGroup()
-include("core.jl")
+serialbenchmarks = [
+    "serial/core.jl",
```

Comment:
use `joinpath`?

Reply:
ah yeah. with `benchdir = dirname(@__FILE__)` and the `joinpath(benchdir, ...)`. yeah that's the way it should be.